README.md

```diff
@@ -27,3 +27,38 @@ If you need to update pip dependencies, after modifying the requirements file, r
 ```
 bazel run //docs:requirements.update
 ```
+
+To update all existing dependencies, run:
+```
+bazel run //docs:requirements.update -- --upgrade
+```
+
+
+### Alternative incremental build
+
+While the above command is the most straightforward and official way to build the documentation,
+it can be slow, especially if you have many minor changes to the documentation and want to see the
+result quickly.
+The generated output will be in the _build directory.
+
+```sh
+bazel run //docs:incremental
+```
+
+### IDE integration
+
+For your IDE to support you with documentation, you need to run the following command.
```

Comment:
What command?

Reply:
It's the command mentioned in line 65. 
`bazel run //docs:ide_support` 

It generates the virtualenv that you then can select in your IDE. 
It will generate a venv called `.venv_docs`

Reply:
Instead of moving it by a few lines, I ended up rewriting the entire documentation. Sorry, please have a look again 🤣